cmd/nerdctl/container_inspect.go

```diff
@@ -38,13 +38,16 @@ func newContainerInspectCommand() *cobra.Command {
 		SilenceErrors:     true,
 	}
 	containerInspectCommand.Flags().String("mode", "dockercompat", `Inspect mode, "dockercompat" for Docker-compatible output, "native" for containerd-native output`)
+	containerInspectCommand.Flags().BoolP("size", "s", false, "Display total file sizes")
```

Comment:
Could you also update https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md#whale-blue_square-nerdctl-inspect

Reply:
Yes.